src/transformState.js

```diff
@@ -62,7 +62,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const obj of transforms) {
```

Comment:
Name your variables properly. 'Obj' doesn't tell anything about your variable. 'Transformation' would fit it better